scripts/delete-old-app-engine-versions.sh

```diff
@@ -148,13 +140,13 @@ case $1 in
 esac
 
 NEW_PROJECT="bvdp-saturn-$1"
-ENV_TO_EXEC="$1"
 
 printf "${INFO} Selected project ${GRN}%s${RST}\n" "${NEW_PROJECT}"
 
 check_user_permissions
 
-DELETION_DATE=$(date --date="-7 days" +%F)
+DELETION_TIME=$(jq -n 'now - 604800') # 604800 seconds = 7 days
```

Comment:
Might be clearer to write out the math
```suggestion
DELETION_TIME=$(jq -n 'now - (7 * 24 * 60 * 60)') # 7 days
```

Reply:
Nice, `jq` really does everything.